### PR TITLE
CLOUDP-93382: Add Programmatic Invite Management for project to the atlas go client

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

Add Programmatic Invite Management for project to the atlas go client

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/go-client-mongodb-atlas/pull/227

File: mongodbatlas/project_invitations.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const projectInvitationBasePath = projectBasePath + "/%s/invites"
+
+// Invitations gets all unaccepted invitations to the specified Atlas project.
+//
+// See more: https://docs-atlas-staging.mongodb.com/cloud-docs/docsworker-xlarge/DOCSP-14695/reference/api/project-get-invitations/
```
Comment (gssbzn) [inline mongodbatlas/project_invitations.go:27]: 👀

**Review by gssbzn — COMMENTED** [review]
Code looks good but there are some links to staging docs

**Review by gssbzn — APPROVED** [review]
LGTM FYI @themantissa

**Pull request merged**